Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -135,30 +188,111 @@ void HiresTexture::Update()
+                " is below the lowest bound. Resetting to lowest allowable value.",
+            20000);
+        WARN_LOG(VIDEO, "Prefetcher: Mem limit cull percent INI override out of "
+                        "lower bound. Setting to lowest");
+      }
+      else if (Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT) > 90.0)
+      {
+        percent_mem_to_cull = 90.0;
+        OSD::AddMessage(
+            "Prefetcher: User INI setting for 'culled percent' of " +
+                StringFromFormat(
+                    "%.1f %%", Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT)) +
+                " is below the highest bound. Resetting to highest allowable value.",
+            20000);
+        WARN_LOG(VIDEO, "Prefetcher: Mem limit cull percent INI override out of "
+                        "higher bound. Setting to highest");
+      }
+    }
+    else
+    {
+      // Successful Manual Override - Dont report values in OSD, routine stats notification will do
+      OSD::AddMessage("Prefetcher: Maximum delegated memory limit has been manually overriden "
+                      "by user INI settings.");
+      percent_mem_to_cull = Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT);
+      INFO_LOG(VIDEO, "Prefetcher: Mem limit overriden by user INI successfully, "
+                      "at %.2f (cull percent)",
+               percent_mem_to_cull);
+    }
+  }
+
+  double prefetched_size_sum = 0;  // total mem size incl. non-native decompressed textures
+  const double absolute_max_mem = Common::MemPhysical();
+  // Currently "total usable", excl. HW reserved
+  const double culled_percent_amount = (Common::MemPhysical() * percent_mem_to_cull) / 100;
+  const double delegated_max_mem = Common::MemPhysical() - culled_percent_amount;
+  const double below_min_req_mem = 3221225472;  // 3 GB; 2GB Rec. + 1GB OS/Misc
+  // WIP - using 1024 * ... etc but makes the line so long, no global ref found, make one?
+
+  // message only
+  if (absolute_max_mem <= below_min_req_mem)
+  {
+    OSD::AddMessage("Prefetcher: Warning! System RAM of " +
+                        StringFromFormat("%.2f MB", absolute_max_mem / (1024.0 * 1024.0)) +
+                        " is below Dolphin's minimum recommended specifications.",
+                    20000);
+    WARN_LOG(VIDEO, "Usable system memory below Dolphin's minimum recommended specifications");
+  }
 
-  size_t size_sum = 0;
-  size_t sys_mem = Common::MemPhysical();
-  size_t recommended_min_mem = 2 * size_t(1024 * 1024 * 1024);
-  // keep 2GB memory for system stability if system RAM is 4GB+ - use half of memory in other cases
-  size_t max_mem =
-      (sys_mem / 2 < recommended_min_mem) ? (sys_mem / 2) : (sys_mem - recommended_min_mem);
-  u32 starttime = Common::Timer::GetTimeMs();
-  for (const auto& entry : s_textureMap)
+  // Routine Stat Notification
```

Comment:
Didn't found better wording yesterday, but left in there as a WIP reminder; could find them today.